js_modules/dagster-ui/packages/ui-core/src/runs/RunTimeline.tsx

```diff
@@ -849,21 +853,21 @@ const RunHoverContent = (props: RunHoverContentProps) => {
                       {run.status === 'SCHEDULED' ? (
                         'Scheduled'
                       ) : (
-                        <Link to={`/runs/${run.id}`}>
-                          <Mono>{run.id.slice(0, 8)}</Mono>
-                        </Link>
+                        <Link to={`/runs/${run.id}`}>{dayjs(run.startTime).fromNow()}</Link>
                       )}
                     </Box>
-                    <Mono>
+                    <div>
                       {run.status === 'SCHEDULED' ? (
                         <TimestampDisplay timestamp={run.startTime / 1000} />
                       ) : (
-                        <TimeElapsed
-                          startUnix={run.startTime / 1000}
-                          endUnix={run.endTime / 1000}
-                        />
+                        <Mono>
```

Comment:
Moved this down so that we're not showing it on the `SCHEDULED` timestamps, just on the elapsed time. (And I'm not even sure we should be doing it there, either.)